temporalcli/commandsgen/commands.yml

```diff
@@ -394,6 +394,162 @@ commands:
         description: Reason for terminating the batch job.
         required: true
 
+  - name: temporal worker-deployment
```

Comment:
@drewhoskins-temporal - I noticed in our gRPC and HTTP API we chose to just call these "deployments" without the term worker at all. I like the "worker" qualifying term, but the inconsistency of not doing so for our API is unfortunate.